### PR TITLE
STORM_B: Change user page in admin and add show

### PR DESCRIPTION
**Pull request by AndriiBuhalo**
image
image
image
Change user page in admin panel:

1. Add ability to edit user name and phone (turbo frame)
2. Add page show for user
3. Cover test
4. Rubocop

### CODE REVIEW

File: app/views/dredd/_header.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <header class="w-full h-16 fixed z-30 bg-light-grey">
  <div class="card-gradient flex items-center justify-between mx-auto px-7 w-full">
     <div class="flex w-1/2 justify-between">
-      <span class="text-2xl text-black"><%= t("dredd.header.admin_dashboard") %></span>
+      <span class="text-2xl text-black"><%= t("dredd.header.admin_dashboard").html_safe %></span>
```
Comment (Shabaldas) [inline app/views/dredd/_header.html.erb:4]: why did you add it?

File: config/locales/en/dredd.en.yml [hunk 2]
```diff
@@ -158,18 +158,20 @@ en:
...
-      workers: WORKERS
+      customers: "CUSTOMERS"
+      customer: "Customer"
+      workers: WORKERS"
       created_at: "CREATED AT"
       nickname: "NICKNAME"
       actions: "ACTIONS"
     header:
-      admin_dashboard: "Admin Dashboard"
+      admin_dashboard: "Admin <br> Dashboard"
```
Comment (Shabaldas) [inline config/locales/en/dredd.en.yml:174]: ????
no no no, any html tags here

File: spec/request/dredd/dashboard_spec.rb [hunk 3]
```diff
@@ -31,7 +31,7 @@
       get dredd_dashboard_path
 
       expect(response).to be_successful
-      expect(response.body).to include('Admin Dashboard')
+      expect(response.body).to include('Admin <br> Dashboard')
```
Comment (Shabaldas) [inline spec/request/dredd/dashboard_spec.rb:34]: same

File: spec/request/dredd/users_spec.rb [hunk 4]
```diff
@@ -53,4 +54,39 @@
...
+      expect(response).to be_successful
+      expect(response.body).to include(user.first_name)
+      expect(response.body).to include(user.last_name)
+      expect(response.body).to include(user.role)
+      expect(response.body).to include(user.phone)
+    end
+  end
+
+  describe 'PUT /dredd/user/:id' do
+    let!(:user_first) { create(:user) }
```
Comment (Shabaldas) [inline spec/request/dredd/users_spec.rb:71]: no needed to call `user_first` here if tou are using one user in your rspec!
you already have a user on a top line:2

File: app/views/dredd/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  <div class="w-full h-[450px] bg-gradient-to-r from-orange-300 via-purple-400 to-orange-300"></div>
+  <div class="absolute top-[50%] left-[50%] transform -translate-x-[50%] -translate-y-[50%] w-[750px] h-[600px] rounded-3xl bg-white drop-shadow-xl">
+    <div class="w-48 h-48 rounded-full absolute -top-24 left-[50%] -translate-x-[50%] bg-gray-100 overflow-hidden">
+      <%= image_tag("dredd/user.png") %>
+    </div>
+    <div class="w-full mt-[100px] h-[500px] grid gap-4 place-content-center text-center">
+      <h3 class="pb-6 text-5xl"><%= @user.full_name %></h3>
+      <div class="p-3 text-2xl"><%= @user.phone %></div>
+      <div class="p-6 text-2xl"><%= @user.email %></div>
+      <div class="p-6 text-2xl"><%= t("dredd.users.#{@user.role}") %></div>
```
Comment (Shabaldas) [inline app/views/dredd/users/show.html.erb:11]: `<%= t("dredd.users.#{@user.role}") %>` 
how does it work, if you did't add any locals to en or ua?

**Review by Shabaldas — CHANGES_REQUESTED** [review]
wrong validation on User model.
for example =>
```ruby
  validates :email, uniqueness: true, on: :create
  ```
  
 You don't need to add custom validation here

File: app/views/dredd/users/show.html.erb (hunk 5 above)
Comment (AndriiBuhalo) [reply app/views/dredd/users/show.html.erb:11]: it is work)
image
image

File: config/locales/en/dredd.en.yml (hunk 2 above)
Comment (AndriiBuhalo) [reply config/locales/en/dredd.en.yml]: done)

File: app/views/dredd/_header.html.erb (hunk 1 above)
Comment (AndriiBuhalo) [reply app/views/dredd/_header.html.erb]: deleted)

File: spec/request/dredd/dashboard_spec.rb (hunk 3 above)
Comment (AndriiBuhalo) [reply spec/request/dredd/dashboard_spec.rb]: done)

File: spec/request/dredd/users_spec.rb (hunk 4 above)
Comment (AndriiBuhalo) [reply spec/request/dredd/users_spec.rb]: done)

**Review by Shabaldas — APPROVED** [review]
💪

**Pull request merged**